ledger/participant-local-store/src/main/scala/com/daml/platform/localstore/api/IdentityProviderConfigStore.scala

```diff
@@ -39,6 +39,20 @@ trait IdentityProviderConfigStore {
       loggingContext: LoggingContext
   ): Future[Boolean]
 
+  final def getActiveIdentityProviderByIssuer(issuer: String)(implicit
+      loggingContext: LoggingContext
+  ): Future[IdentityProviderConfig] =
+    getIdentityProviderConfig(issuer)
+      .flatMap {
+        case Right(value) if !value.isDeactivated => Future.successful(value)
+        case Right(value) =>
+          Future.failed(
+            new Exception(s"Identity Provider ${value.identityProviderId.value} is deactivated.")
```

Comment:
Use error codes?

Reply:
Actually, I have a question here :) This exception is not thrown to the gRPC layer, to the user - this is covered by the caller code which is parsing a token and gives result. In the end nothing is thrown to the user, do you still think we need to throw an error code here?

Reply:
Some error codes appear just in the participant server logs and are useful to participant operators, so by default I'd use error codes.

Are you sure this won't be transmitted to the client? Who catches/handles it then?

Reply:
i have added a comment there

Reply:
Forgot to mention as well for others - we spoke about this case offline